services/src/Microsoft.Azure.IIoT.Services.OpcUa.Publisher/src/Startup.cs

```diff
@@ -223,6 +223,7 @@ public class Startup {
                 .AsImplementedInterfaces().SingleInstance();
             builder.RegisterType<CosmosDbServiceClient>()
                 .AsImplementedInterfaces();
+            // TODO check why duplicate?
```

Comment:
Please verify why we have this before check in and remove the TODO

Reply:
ok, I removed the duplicated code